src/test/e2e/20_zarf_init_test.go

```diff
@@ -18,86 +19,84 @@ import (
-	require.NoError(t, err)
-	stateJSON, err := base64.StdEncoding.DecodeString(base64State)
-	require.NoError(t, err)
-	state := types.ZarfState{}
-	err = json.Unmarshal(stateJSON, &state)
-	require.NoError(t, err)
-	require.NotContains(t, logText, state.AgentTLS.CA)
-	require.NotContains(t, logText, state.AgentTLS.Cert)
-	require.NotContains(t, logText, state.AgentTLS.Key)
-	require.NotContains(t, logText, state.ArtifactServer.PushToken)
-	require.NotContains(t, logText, state.GitServer.PullPassword)
-	require.NotContains(t, logText, state.GitServer.PushPassword)
-	require.NotContains(t, logText, state.RegistryInfo.PullPassword)
-	require.NotContains(t, logText, state.RegistryInfo.PushPassword)
-	require.NotContains(t, logText, state.RegistryInfo.Secret)
-	require.NotContains(t, logText, state.LoggingSecret)
-
-	// Check that gitea is actually running and healthy
-	stdOut, _, err = e2e.ExecZarfCommand("tools", "kubectl", "get", "pods", "-l", "app in (gitea)", "-n", "zarf", "-o", "jsonpath={.items[*].status.phase}")
-	require.NoError(t, err)
-	require.Contains(t, stdOut, "Running")
-
-	// Check that the logging stack is actually running and healthy
-	stdOut, _, err = e2e.ExecZarfCommand("tools", "kubectl", "get", "pods", "-l", "app in (loki)", "-n", "zarf", "-o", "jsonpath={.items[*].status.phase}")
-	require.NoError(t, err)
-	require.Contains(t, stdOut, "Running")
-	stdOut, _, err = e2e.ExecZarfCommand("tools", "kubectl", "get", "pods", "-l", "app.kubernetes.io/name in (grafana)", "-n", "zarf", "-o", "jsonpath={.items[*].status.phase}")
-	require.NoError(t, err)
-	require.Contains(t, stdOut, "Running")
-	stdOut, _, err = e2e.ExecZarfCommand("tools", "kubectl", "get", "pods", "-l", "app.kubernetes.io/name in (promtail)", "-n", "zarf", "-o", "jsonpath={.items[*].status.phase}")
-	require.NoError(t, err)
-	require.Contains(t, stdOut, "Running")
-
-	// Check that the registry is running on the correct NodePort
-	stdOut, _, err = e2e.ExecZarfCommand("tools", "kubectl", "get", "service", "-n", "zarf", "zarf-docker-registry", "-o=jsonpath='{.spec.ports[*].nodePort}'")
-	require.NoError(t, err)
-	require.Contains(t, stdOut, "31337")
-
-	// Check that the registry is running with the correct scale down policy
-	stdOut, _, err = e2e.ExecZarfCommand("tools", "kubectl", "get", "hpa", "-n", "zarf", "zarf-docker-registry", "-o=jsonpath='{.spec.behavior.scaleDown.selectPolicy}'")
-	require.NoError(t, err)
-	require.Contains(t, stdOut, "Min")
-
-	// Special sizing-hacking for reducing resources where Kind + CI eats a lot of free cycles (ignore errors)
-	_, _, _ = e2e.ExecZarfCommand("tools", "kubectl", "scale", "deploy", "-n", "kube-system", "coredns", "--replicas=1")
-	_, _, _ = e2e.ExecZarfCommand("tools", "kubectl", "scale", "deploy", "-n", "zarf", "agent-hook", "--replicas=1")
+	t.Run("init mismatched arch", func(t *testing.T) {
```

Comment:
We should test this logic in the test specifically designed around mismatched architectures <3